cfpb/erap-lookup
# Create react app and display program data

Create react app and add/display unfiltered program data.

## Additions

- React app
- Program data
- Notification module
- Results module

## Testing

1. Check out branch
2.  Run `yarn start`
3. Verify erap program results are displayed in page
